### PR TITLE
master-qa-118771

### PR DESCRIPTION
**Pull request by zhangyan0701**
https://issues.liferay.com/browse/LPS-118771

### CODE REVIEW

**Comment by zhangyan0701** [comment]
ci:test:sf

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by john-co** [comment]
Just started reviewing :)

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro [hunk 1]
```diff
@@ -230,13 +229,12 @@ definition {
...
 		SelectFrame(locator1 = "IFrame#MODAL_BODY");
 
-		var key_cardText = "${themeName}";
-
 		AssertTextPresent(
-			locator1 = "Card#SPECIFIC_LARGE_TEXT",
-			value1 = "${key_cardText}");
+			key_card = "${themeName}",
+			locator1 = "LexiconCard#CARD",
+			value1 = "${themeName}");
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro:235]: :clap: I like seeing this!

Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro:235]: Did you try LexiconCard.viewCardTitle? Not sure which viewCard* macro would match the themeName on the card.

Comment (zhangyan0701) [reply portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro:235]: I didn't, I didn't see this macro, I will try later.

**Comment by john-co** [comment]
cc/ @zhangyan0701 please see comments. That's great that we can consolidate the Card.path and LexiconCard.path. The other thing, I would like to see is actually having the LexiconCard.path renamed to Card.path since there's no mention of lexicon in the card element besides the icons. Same thing for the other Lexicon*.paths. Otherwise, the other UISections and UIElements could end up having clay or lexicon in front of them as well.

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro (hunk 1 above)
Comment (zhangyan0701) [reply portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro:235]: LexiconCard.viewCardTitle works for this case. Will resend you a PR.

**Comment by zhangyan0701** [comment]
Yes, that make sense.

**Comment by zhangyan0701** [comment]
ci:test:sf

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by john-co** [comment]
Looks good!

**Comment by john-co** [comment]
ci:forward

**Pull request closed without merging**